bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/health/SwitchOfHealthCheckCommand.java

```diff
@@ -0,0 +1,128 @@
+ * under the License.
+ */
+package org.apache.bookkeeper.tools.cli.commands.health;
+
+import com.beust.jcommander.Parameter;
+import com.google.common.util.concurrent.UncheckedExecutionException;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.apache.bookkeeper.conf.ServerConfiguration;
+import org.apache.bookkeeper.meta.LedgerUnderreplicationManager;
+import org.apache.bookkeeper.meta.MetadataDrivers;
+import org.apache.bookkeeper.meta.exceptions.MetadataException;
+import org.apache.bookkeeper.meta.zk.ZKMetadataBookieDriver;
+import org.apache.bookkeeper.meta.zk.ZKMetadataDriverBase;
+import org.apache.bookkeeper.replication.ReplicationException;
+import org.apache.bookkeeper.tools.cli.helpers.BookieCommand;
+import org.apache.bookkeeper.tools.framework.CliFlags;
+import org.apache.bookkeeper.tools.framework.CliSpec;
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.zookeeper.KeeperException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.ExecutionException;
+
+
+/**
+ * Command to enable or disable auto recovery in the cluster.
+ */
+public class SwitchOfHealthCheckCommand extends BookieCommand<SwitchOfHealthCheckCommand.HealthCheckFlags> {
+
+    static final Logger LOG = LoggerFactory.getLogger(SwitchOfHealthCheckCommand.class);
+
+    private static final String NAME = "switch";
+    private static final String DESC = "Enable or disable health check in the cluster. Default is enable.";
+
+    public SwitchOfHealthCheckCommand() {
+        this(new HealthCheckFlags());
+    }
+
+    private SwitchOfHealthCheckCommand(HealthCheckFlags flags) {
+        super(CliSpec.<HealthCheckFlags>newBuilder()
+                .withName(NAME).withDescription(DESC)
+                .withFlags(flags).build());
+    }
+
+    /**
+     * Flags for health check command.
+     */
+    @Accessors(fluent = true)
+    @Setter
+    public static class HealthCheckFlags extends CliFlags {
+
+        @Parameter(names = { "-d", "--disable" }, description = "Enable or disable health check.")
+        private boolean disable;
+
+        @Parameter(names = {"-s", "--status"}, description = "Check the health check status.")
+        private boolean status;
+
+    }
+
+    @Override
+    public boolean apply(ServerConfiguration conf, HealthCheckFlags cmdFlags) {
+        try {
+            return handler(conf, cmdFlags);
+        } catch (MetadataException | ExecutionException e) {
+            throw new UncheckedExecutionException(e.getMessage(), e);
+        }
+    }
+
+    private boolean handler(ServerConfiguration conf, HealthCheckFlags flags)
+            throws MetadataException, ExecutionException {
+
+        MetadataDrivers.runFunctionWithMetadataBookieDriver(conf, driver -> {
+            try {
+                String enableHealthPath = conf.getEnableHealthPath();
+
+                if(!(driver instanceof ZKMetadataBookieDriver)){
```

Comment:
please remove this line, you can add a dummy implementation in the MetadataBookieDriver (using "default" methods)